substrate/frame/support/src/tests/mod.rs

```diff
@@ -227,7 +227,7 @@ crate::construct_runtime!(
 	}
 );
 
-#[crate::derive_impl(self::frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
+#[crate::derive_impl(self::frame_system::config_preludes::TestDefaultConfig as self::frame_system::DefaultConfig)]
```

Comment:
Still need `as` here?

Reply:
Yeah, as the trait is scoped under `self::frame_system` and not `self` as expected by the macro for default behaviour.